dist/legacy-assets/js/main.js

```diff
@@ -295,6 +308,9 @@ function getPageData(collectionId, path, success, error) {
     dataType: 'json',
     type: 'GET',
     success: function (response) {
+      var date = new Date();
+      date = date.getHours() + ":" + date.getMinutes() + "." + date.getMilliseconds();
```

Comment:
Perhaps worth doing if we use it more often? We haven't got any global logging function yet but could be worth adding it into something like that if we find ourselves putting more time into it.